trieste/acquisition/function/function.py

```diff
@@ -762,3 +763,79 @@ def acquisition(x: TensorType) -> TensorType:
         return tf.exp(tf.linalg.logdet(covariance + jitter))
 
     return acquisition
+
+
+class BayesianActiveLearningByDisagreement(SingleModelAcquisitionBuilder):
+    """
+    Builder for the *Bayesian Active Learning By Disagreement* acquisition function defined in
+    :cite:`houlsby2011bayesian`. The acquisition function computes the information gains
+    of the predictive entropy.  
+
+    This acquisition function is intended to use for Binary Gaussian Process Classification
+    model with Bernoulli likelihood. integrating over nuisance parameters is not currently 
```

Comment:
"Integrating over nuisance parameters is currently not supported (see equation 6 of the paper)."